### ISSUE TITLE
About the dataset replacement

### ISSUE THREAD

**Issue by 3232731490** [SebastianGer/WildfireSpreadTS#1]
Can I use the model you provide in this repository to train next_day_wildfire_spread dataset? What changes do I need to make if so?

**Comment by SebastianGer** [comment]
> Can I use the model you provide in this repository to train next_day_wildfire_spread dataset? What changes do I need to make if so?

EDIT: I misunderstood your question at first. I assumed you wanted to combine the two datasets. 

You should be able to train the models on that dataset. The UTAE and ConvLSTM would not be useful though, since they assume that you have multiple days as input, which NextDayWildfireSpread does not provide, to the best of my knowledge. The other models are a logistic regression and a UNet with a ResNet encoder, which are not very interesting architectures and nothing about them is specialized to the fire application. 

The main contribution of this repository is to provide baselines for our multi-temporal active fire spread dataset, not to provide innovation in the area of model architectures. We only used existing architectures that were used in a lot of other papers already.

**Comment by 3232731490** [comment]
Thank you for your reply, I have an overview of the situation, I recently used pytorch to write a model training next_day_wildfire_spread dataset, but the effect is not good, so I want to try to use the model code you provided to see if the effect will improve.

However, I changed the model code a little to make it work properly to train next_day_wildfire_spread dataset, but similar to the model training results I wrote, the train_loss and val_loss almost fluctuate in a small interval (stable at about 0.67, which cannot be optimized).

I would like to ask what is the cause of this problem, is it because the loss function is not selected well or is it a problem with the model structure?

**Comment by SebastianGer** [comment]
> Thank you for your reply, I have an overview of the situation, I recently used pytorch to write a model training next_day_wildfire_spread dataset, but the effect is not good, so I want to try to use the model code you provided to see if the effect will improve.
> 
> However, I changed the model code a little to make it work properly to train next_day_wildfire_spread dataset, but similar to the model training results I wrote, the train_loss and val_loss almost fluctuate in a small interval (stable at about 0.67, which cannot be optimized).
> 
> I would like to ask what is the cause of this problem, is it because the loss function is not selected well or is it a problem with the model structure?

Have you compared your results to the original paper's results in terms of the metrics they used? It might be that you are very quickly reaching a somewhat good solution and then not much is happening afterwards. Their results were relatively good when they only used vegetation and fire as input (Table V), so if your model learns to ignore everything but those features at the beginning, it would make sense that the loss only mildly fluctuates afterwards.
